dataprofiler/data_readers/csv_data.py

```diff
@@ -507,7 +508,10 @@ def _load_data_from_file(self, input_file_path):
         Loads the data into memory from the file.
         """
         
-        self._file_encoding = data_utils.detect_file_encoding(input_file_path)
+        self._file_encoding = None
+        if not data_utils.is_stream_buffer(input_file_path) or isinstance(input_file_path, BytesIO):
```

Comment:
make sure all lines are < 80 chars

Reply:
If `is_stream_buffer` returns True when the file path is a StringIO or BytesIO, can't this if condition be simplified to just `if not isinstance StringIO`?

Reply:
Also FYI I'm looking at the is_stream_buffer on my branch right now and its docstring is out of date and lists a bunch of parameters that are no longer in the function signature, you should correct that in this PR if possible.